modules/installation-azure-user-defined-routing.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-private.adoc
+
+[id="installation-azure-user-defined-routing_{context}"]
+= User-defined outbound routing
+
+In {product-title}, you can choose your own outbound routing for a cluster to
+connect to the internet. This allows you to skip the creation of public IP
+addresses and the public load balancer.
+
+You can configure user-defined routing by modifying parameters in the
+`install-config.yaml` file before installing your cluster. A pre-existing VNet
+is required to use outbound routing when installing a cluster; the installation
+program is not responsible for configuring this.
+
+When configuring a cluster to use user-defined routing, the installation program
+does not create the following resources:
+
+* Outbound rules for access to Internet.
```

Comment:
to Internet -> to the Internet

Reply:
Yeah, I wasn't sure if this was a "thing" with *Internet* in OCP. I saw several other places referring to it as uppercase with no *the*, and also had several devs refer to it in that way :sweat_smile: . I'll go with the lowercase version with "the", since that seems the most natural.